src/scripts.js

```diff
@@ -47,33 +84,27 @@ function viewFavorites() {
     user.favoriteRecipes.forEach(recipe => {
       cardArea.insertAdjacentHTML('afterbegin', `<div id='${recipe.id}'
       class='card'>
-      <header id='${recipe.id}' class='card-header'>
+      <header data-id='${recipe.id}' class='card-header'>
```

Comment:
I think data-id is a good idea to use instead of how it was before.  We were always told that id's should be unique to only one element while dataset id's can belong to multiple, so that was a good idea changing all those instances